src/mqtt/protocol/mqtt/mqtt_quic_client.c

```diff
@@ -1749,6 +1749,12 @@ mqtt_quic_ctx_send(void *arg, nni_aio *aio)
 			if (pub_pipe == NULL) {
 				log_info("create new pub stream");
 				nni_lmq_put(s->topic_lmq, msg);
+				/*
+				 * You can set dialer quic priority before
+				 * dialer_start to create a stream with the
+				 * specified priority.
+				 */
+				//nni_dialer_setopt(npipe->p_dialer, NNG_OPT_MQTT_QUIC_PIRORITY, &val, sizeof(int), NNI_TYPE_INT32);
```

Comment:
Is the priority property just for publish stream?

Reply:
For now, only publish streams.

But it can get complicated later, so this is just exposing the api. At the same time, I did some tests, but the test effect is not very significant.

Reply:
In my opinion. Subscribe request should have a priority that is not too low. (It could be a fixed priority value)